include/dem/data_containers.h

```diff
@@ -221,7 +221,11 @@ namespace DEM
 
     // [cell iterators]
     typedef std::vector<typename Triangulation<dim>::active_cell_iterator>
-      cell_container;
+      cell_vector;
+
+    // [cell iterators]
+    typedef std::set<typename Triangulation<dim>::active_cell_iterator>
+      cell_set;
```

Comment:
Why not `std::unordered_set`?

Reply:
We don't have a hash function for active_cell_iterator right now so we cannot use unordered set. I'll work on fixing this in the mid-term, but for now we can only use sets